cordapp-cpk2/src/main/java/net/corda/plugins/cpb2/CpbTask.java

```diff
@@ -59,11 +68,21 @@ public CpbTask() {
             m.getAttributes().put(CPB_NAME_ATTRIBUTE, getArchiveBaseName());
             m.getAttributes().put(CPB_VERSION_ATTRIBUTE, getArchiveVersion());
         });
+
+        try {
+            jarDir = Files.createTempDirectory("").toFile();
+            jarDir.deleteOnExit();
+        } catch (IOException e) {
+            getLogger().warn("Could not create jar directory: {}", e.getMessage());
+            jarDir = null;
+        }
```

Comment:
I think we should re-throw the exception or remove the try/catch so the original exception is allowed to be thrown because these changes are needed for correct behaviour of the plugin. If we drop the exception and continue, we are allowing the incorrect behaviour we are trying to fix to continue happening in this error path.

Reply:
I've changed it so it fails with the error now